Uml/Uml.csproj

```diff
@@ -5,13 +5,19 @@
     <Company>RHEA System S.A.</Company>
     <Title>UML for dot net</Title>
     <Version>0.1.0</Version>
+    <AssemblyVersion>0.1.0</AssemblyVersion>
+    <FileVersion>0.1.0</FileVersion>
     <Description>A C# implementation of the UML 2.5.1 model</Description>
     <Copyright>Copyright © RHEA System S.A.</Copyright>
     <Authors>Sam Gerené, Alex Vorobiev</Authors>
     <PackageId>uml4net</PackageId>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <RepositoryType>Git</RepositoryType>
     <RepositoryUrl>https://github.com/RHEAGROUP/uml4net</RepositoryUrl>
+    <DebugType>full</DebugType>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>

```

Comment:
these three "Generate..." elements are not required since we don't have these attrributes specified in the AssemblyInfo.cs file. They should be removed.